### ISSUE TITLE
[Compat?? Bug] - Collapsible Glider Enchantments

### ISSUE THREAD

**Issue by AriaElidove** [BluSunrize/ImmersiveEngineering#5968]
### Minecraft Version

1.20.4

### Immersive Engineering Version

11.4.1-176

### Expected Behavior

I expected the Collapsible Glider to accept enchantments **OR** EMI to not show that it can be enchanted.

### Actual Behavior

EMI shows that the Collapsible Glider is able to be enchanted with Curse of Vanishing, Curse of Binding, Mending, and Unbreaking 3 (yet not 1 or 2 for some reason???)

### Steps to reproduce

Check the uses for the Collapisble Glider with EMI
Look at Anvil Repairing tab in EMI
Enchantments
Try to enchant the Glider with an enchanted book of one of the previous enchantments; Mending Book, for example
Fail to enchant and cry

### Debug Crash Log

https://gist.github.com/AriaElidove/da08f5e017e01294d84fc1268edc9a8d

**Comment by AriaElidove** [comment]
Adding onto this, some items may follow this issue; Wire Cutters, for example.

**Comment by BluSunrize** [comment]
In my own testing with JEI, this doesn't happen. So I'm tempted to put this on EMI instead of us ^^

**Comment by voidsong-dragonfly** [comment]
I would agree this is on EMI's end, they seem to be relying on enchantment table methods in a way that doesn't seem right to me. `Item#isEnchantable(...)` shouldn't be used for determining if an item can take an enchantment.

Regardless I can probably add some extra checks to make sure no mod thinks IE items of specific flavors are enchantable.
